[IMP] accounting: update the analytic budgets page (#11330)
odoo/documentation: content/applications/finance/accounting/reporting/budget.rst

=======
Budgets
=======

:ref:`Analytic budgets <accounting/reporting/budgets/analytic-budgets>` track and forecast
specific activities, helping businesses make informed decisions about internal operations. In
contrast, :ref:`financial budgets <accounting/reporting/budgets/financial>`, are tied to the
general ledger and focus on the company’s overall economic position.

.. _accounting/reporting/budgets/analytic-budgets:

Analytic Budgets
================

Analytic budgets allow for allocating and tracking income and expenses in detail, breaking down

> costs and revenues by specific projects, departments, or activities. Analytic budgets can be applied

Author: @dade-odoo you're right, it's more precise this way ;)

across various departments or projects to measure profitability and performance. Odoo manages
analytic budgets using :doc:`analytic accounting <analytic_accounting>`.

.. _accounting/reporting/budgets/analytic-configuration:

Configuration
-------------

Go to :menuselection:`Accounting --> Configuration --> Settings`, and enable
:guilabel:`Budget Management` in the :guilabel:`Analytics` section.

.. _accounting/reporting/budgets/analytic-accounting:

Analytic accounting
-------------------

> Odoo structures budgets using :ref:`accounts <analytic_accounting/analytic_plans>` and

Author: @dade-odoo Good catch, thanks!

lines are settled, click :guilabel:`Open`.
If changes need to be done once the budget is in the :guilabel:`Open` status, there are two options:

- :guilabel:`Reset to Draft`: To overwrite the data, then reopen the budget.
- :guilabel:`Revise`: A new budget will be created. Once it is set to the :guilabel:`Open` status,
  Odoo will add a :guilabel:`Rev` reference to the :guilabel:`Budget name`. The original budget has
  then the :guilabel:`Revised` status.

.. _accounting/reporting/budgets/analytic-budget-check:

Check an analytic budget
------------------------

Once the budget is in the :guilabel:`Open` status, two additional columns are available:
:guilabel:`Committed` and :guilabel:`Achieved`. Their amounts are automatically calculated based on
the vendor bills/invoices and their related analytic distribution. They evolve when a new journal
entry linked to the analytic account is created. The :guilabel:`Achieved` amount reflects the
current result according to the expenses or income for the associated analytic account. In contrast,
the :guilabel:`Committed` amount displays what has already been purchased or sold, regardless of

> whether it has been billed or invoiced.

Author: @dade-odoo Yes, it's more exhaustive this way ;) Thanks for adding all the details to complete it!

.. note::
   For budgets in the :guilabel:`Open` status, when a purchase order created with the associated
   analytic distribution exceeds the amount budgeted, the related purchase order line turns red.

To reveal the :guilabel:`Theoretical` amount or percentage, use the :icon:`oi-settings-adjust`
(:guilabel:`settings adjust`) icon in the far right of the budget top row. The
:guilabel:`Theoretical` represents the amount of money that could theoretically have been spent or
should have been received based on the date.

> To open the reporting view, click :guilabel:`Details` and filter the budget lines and columns of

Author: @dade-odoo Yes, good idea! Is it ok if the Budget Report is not the "full version" this way, as there is only the information related to the selected line of the budget? What do you think?

> To open the reporting view, click :guilabel:`Details` and filter the budget lines and columns of

Reviewer: Great point, let's try: 

> Clicking :guilabel:`Details` opens a filtered view of the :ref:`budget report<link-goes-here>` related to that specific budget line.

> To open the reporting view, click :guilabel:`Details` and filter the budget lines and columns of

Author: @dade-odoo Great! That rocks :)

the pivot table.

.. image:: budget/budget.png

>    :alt: open budget with committed, achieved and theoretical amounts

Author: 😄

.. note::
   Deletion of a budget is only allowed in the :guilabel:`Draft` and :guilabel:`Cancelled` stages.

To view all the different budget lines directly from the :guilabel:`Budgets` view, select the
:guilabel:`Budget Name` and click :guilabel:`Budget Lines`.

.. _accounting/reporting/budgets/analytic-budget-generate:

Generate periodical budgets
~~~~~~~~~~~~~~~~~~~~~~~~~~~

The :guilabel:`Generate` option allows the creation of multiple budget periods. To do so, select the

> budget and click :guilabel:`Generate` to open the :guilabel:`Generate Budget` window. Set the dates,

Author: @dade-odoo haha, I didn't even know you could use this button without selecting a budget 😄 thanks! 
I see that this feature includes all the accounts from the plan selected when creating these periodic budgets. So it's a problem if you select an existing budget because it doesn't take it into account nor does it split the amounts already in there. So I'll get rid of all this and start it from scratch ;)

select the :guilabel:`Period` and the :guilabel:`Analytic Plans`, then click :guilabel:`Split`.

.. image:: budget/generate-budgets.png
   :alt: all the options to generate periodical budgets

.. _accounting/reporting/budgets/analytic-budget-reporting:

Reporting
---------

To perform various reporting actions, go to :menuselection:`Accounting --> Reporting -->
Budget Report`, then:

- Track, analyze, and compare budget data.
- Filter and group Data using the :icon:`fa-plus-square` (:guilabel:`plus-square`) or
  :icon:`fa-minus-square` :guilabel:`(minus-square)` icon.
- Drill down into the report to see more detail on the actual amounts and transactions.
- Export for further analysis or reporting needs.

.. _accounting/reporting/budgets/financial:

Financial budgets
=================

> Financial budgets, available in the :guilabel:`Profit and Loss` report, are structured around

Author: @dade-odoo thanks for the explanation, luv it!

specific accounts and transactions for official financial reporting and compliance purposes.

.. _accounting/reporting/budgets/financial-budget-set:

Set a financial budget
----------------------

To create a new financial budget, follow these steps:

#. Go to :menuselection:`Accounting --> Reporting --> Profit and Loss`, and open the
   :ref:`Profit and Loss <accounting/reporting/profit-and-loss>` report.
#. Use the :icon:`fa-calendar` :guilabel:`(calendar)` icon to use the date selector and choose a
   period.
#. Click the :icon:`fa-bar-chart` :guilabel:`Budget` icon and name the budget. A new column labeled

>    with the budget name appears next to the :guilabel:`Balance` column.

Reviewer: And maybe this should say :icon:`fa-bar-chart` :guilabel:`(bar chart)` :guilabel:`Budget` button? I'm not sure. We can confirm with Audrey/Xavier when it goes through them :)

>    with the budget name appears next to the :guilabel:`Balance` column.

Author: @dade-odoo I don't think you should add it when it's the name of a menu ... we'll see

#. Assign amounts to each account that needs analysis.

> #. A new :guilabel:`%` column appears to the right of the new budget column, indicating the current
>    status.

Reviewer: ```suggestion
#. A new :guilabel:`%` column will appear to the right of the new budget column, indicating the
   current status.
```

  Odoo will add a :guilabel:`Rev` reference to the :guilabel:`Budget Name`. The original budget then
  has the :guilabel:`Revised` status.

.. _accounting/reporting/budgets/analytic-budget-check:

Check an analytic budget
------------------------

Once the budget is in the :guilabel:`Open` status, two additional columns are available:
:guilabel:`Committed` and :guilabel:`Achieved`. These columns' amounts are automatically calculated
based on the related analytic distribution of journal items. When the analytic distribution of a
journal item within the budget's period is updated, the budget's columns for the analytic account(s)
selected in the distribution update automatically. The :guilabel:`Achieved` amount reflects the
current result according to the items of confirmed journal entries for the associated analytic
account. In contrast, the :guilabel:`Committed` amount displays the full value of the
:guilabel:`Achieved` amount, plus any confirmed sales or purchase orders that have not yet been
invoiced or billed.

.. note::
   - When a line in a request for quotation or purchase order includes an analytic distribution, a

>      :guilabel:`Budget` smart button appears, providing a link to the budget report for more

Author: @auva-odoo You are not 😄, there is a point!

.. _accounting/reporting/budgets/analytic-budget-generate:

Generate periodic budgets
-------------------------

The :guilabel:`Generate` button allows the creation of periodic budgets (monthly, quarterly, and
yearly) for the selected :guilabel:`Analytic Plans`. Odoo generates a new budget for each
:guilabel:`Period` between the start and end dates:

- If a single analytic plan is selected, each budget includes a line for each account in that
  analytic plan.
- If multiple analytic plans are selected, each budget includes a line for each unique combination
  involving one account per analytic plan.

To generate periodic budgets, follow these steps:

#. In the :guilabel:`Budgets` list view, click :guilabel:`Generate`.
#. In the :guilabel:`Generate Budget` window, set the dates and select the :guilabel:`Period` and
   the :guilabel:`Analytic Plans`.

   .. image:: budget/generate-budgets.png
      :alt: all the options to generate periodical budgets

#. Click :guilabel:`Split` to create the periodic budgets.
#. Click :guilabel:`Budgets` in the top-left corner to return to the :guilabel:`Budgets` list view.

> #. One by one, click on the different periodic budgets in :guilabel:`Draft` status to open them and

Reviewer 2: ```suggestion
#. One by one, click on the different periodic budgets with the :guilabel:`Draft` status to open them and
```

   set the amounts in the :guilabel:`Budgeted` column for each analytic account linked to the
   chosen analytic plans.
#. Click :guilabel:`Open` for each periodic budget.

.. _accounting/reporting/budgets/analytic-budget-reporting:

Reporting
---------

To perform various reporting actions, go to :menuselection:`Accounting --> Reporting -->
Budget Report`, then:

- Track, analyze, and compare budget data.
- Filter and group data using the :icon:`fa-plus-square` (:guilabel:`plus-square`) or
  :icon:`fa-minus-square` :guilabel:`(minus-square)` icon.

> - Drill down into the report to see more detail on the actual amounts and transactions.

Reviewer 2: ```suggestion
- Drill down into the report to see more details on the actual amounts and transactions.
```
